app/src/main/java/com/realityexpander/tasky/presentation/agenda_screen/AgendaEvent.kt

```diff
@@ -0,0 +1,44 @@
+package com.realityexpander.tasky.presentation.agenda_screen
+
+import com.realityexpander.tasky.presentation.util.UiText
+
+sealed class AgendaEvent {
```

Comment:
Could also be a `sealed interface` if it doesn't need any constructor arguments. That way it'll occupy less memory.

Reply:
Pretty neat. This is my first time using a sealed interface. Very interesting.

OK, i will implement this!